readout.sh

```diff
@@ -23,16 +23,16 @@ case $ARCHITECTURE in
     osx*) [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost);;
 esac
 
-cmake $SOURCEDIR                                              \
-      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
-      ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}              \
-      ${COMMON_O2_VERSION:+-DCommon_ROOT=$COMMON_O2_ROOT}     \
-      ${MONITORING_VERSION:+-DMonitoring_ROOT=$MONITORING_ROOT} \
+cmake $SOURCEDIR                                                         \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                                \
+      ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}                         \
+      ${COMMON_O2_VERSION:+-DCommon_ROOT=$COMMON_O2_ROOT}                \
+      ${MONITORING_VERSION:+-DMonitoring_ROOT=$MONITORING_ROOT}          \
       ${CONFIGURATION_VERSION:+-DConfiguration_ROOT=$CONFIGURATION_ROOT} \
-      ${READOUTCARD_VERSION:+-DReadoutCard_ROOT=$READOUTCARD_ROOT} \
-      ${INFOLOGGER_VERSION:+-DInfoLogger_ROOT=$INFOLOGGER_ROOT} \
-      ${FAIRROOT_VERSION:+-DFAIRROOTPATH=$FAIRROOT_ROOT}      \
-      ${FAIRROOT_VERSION:+-DFairRoot_DIR=$FAIRROOT_ROOT}      \
+      ${READOUTCARD_VERSION:+-DReadoutCard_ROOT=$READOUTCARD_ROOT}       \
+      ${LIBINFOLOGGER_VERSION:+-DInfoLogger_ROOT=$LIBINFOLOGGER_ROOT}    \
```

Comment:
Is this correct? Looking at the libInfoLogger recipe, it looks like it's still exporting `INFOLOGGER_ROOT` instead of `LIBINFOLOGGER_ROOT`.

Reply:
Yes. `LIBINFOLOGGER_ROOT` is what alibuild generates automatically at build time and it's not available at runtime.